/**:
  ros__parameters:
    forward_path_length: 1000.0
    backward_path_length: 5.0
    stop_line_extend_length: 5.0
    max_accel: -2.8
    max_jerk: -5.0
    system_delay: 0.5
    delay_response_time: 0.5
    is_publish_debug_path: false # publish all debug path with lane id in each module
